File: tests/fixtures/sample.ts
import { readFile } from "fs/promises";
import path from "path";

export function greet(name: string): string {
  return `Hello, ${name}`;
}

export const formatDate = (date: Date): string => {
  return date.toISOString();
};

function internal() {
  greet("world");
  formatDate(new Date());
}

export class UserService {
  getUser(id: string) {
    return readFile(path.join("users", id));
  }
}

File: tests/fixtures/sample.py
import os
from pathlib import Path

def greet(name: str) -> str:
    return f"Hello, {name}"

def _internal_helper():
    pass

class UserService:
    def get_user(self, user_id: str):
        greet(user_id)
        return Path(os.getcwd()) / user_id
